uwrt_mars_rover_utils/uwrt_mars_rover_xbox_controller/include/uwrt_mars_rover_xbox_controller/coordinateNode.hpp

```diff
@@ -0,0 +1,25 @@
+#ifndef COORDINATENODE_HPP_
+#define COORDINATENODE_HPP_
+
+#include <uwrt_mars_rover_xbox_controller/visibility.h>
+
+#include <geometry_msgs/msg/twist.hpp>
+#include <rclcpp/rclcpp.hpp>
+#include <uwrt_mars_rover_xbox_controller/msg/xbox_controller.hpp>
+
+namespace drivetraincontrollerComposition
+{
+class CoordinateNode : public rclcpp::Node
+{
+public:
+  UWRT_MARS_ROVER_XBOX_CONTROLLER_PUBLIC
+  explicit CoordinateNode(const rclcpp::NodeOptions & options);
+
+private:
+  rclcpp::Publisher<geometry_msgs::msg::Twist>::SharedPtr pub_;
+  rclcpp::Subscription<uwrt_mars_rover_xbox_controller::msg::XboxController>::SharedPtr
+    drvtrain_joy_sub;
```

Comment:
This isn't necessarily only for the drivetrain so call it xbox_controller instead. Also, put an underscore after variable names to indicate that they are private to the class (good coding practice in C++). 

```suggestion
  rclcpp::Subscription<uwrt_mars_rover_xbox_controller::msg::XboxController>::SharedPtr
    xbox_controller_sub_;
```